source/_posts/2016-08-05-community-manager.md

```diff
@@ -0,0 +1,29 @@
+---
+title: IIIF Community and Communications Officer Hired
+author: Gabrielle Karampelas
+date: 2016-08-05
+tags: [specifications, image-api, presentation-api, search-api, announcements]
+layout: post
+---
+
+Sheila Rabun has been appointed the new Community and Communications Officer for the International Image Interoperability Framework (IIIF) community; she will start in this new role at the end of August, 2016.
+
+IIIF has seen remarkable growth since June 2015, when 11 institutions joined together to formally organize the [IIIF Consortium][iiif-c] (IIIF-C). There has been rapid evolution in its specifications, IIIF-compatible software, institutions with interoperable content, and community members. Rabun's is the first dedicated staff position for the global community, and will focus on coordinating the expanding suite of IIIF features and functionality, facilitating communication within and beyond the current community, and helping implementers and individual users engage in the Framework.
+
+"I am excited to join the IIIF community and build upon the excellent accomplishments of the initiative thus far," Rabun said in a written statement. Formerly the Digital Project Manager and Interim Director of the University of Oregon Libraries' Digital Scholarship Center, Rabun managed the development of the [Oregon Digital Hydra][oregondigital] repository, the [Oregon Digital Newspaper Program][odnp] (ODNP) and [Historic Oregon Newspapers online][oregonnews], the collaborative [Open Online Newspaper Initiative][open-oni] (Open ONI) software development project, and a variety of [digital scholarship projects][uoregon].
+
+"I am passionate about digital literacy and encouraging the use of digital applications to conduct research, analysis, and dissemination as an extension of traditional academia, and I look forward to strengthening internal and external communication within the IIIF Consortium and broader community, organizing documentation and training efforts, and building new partnerships through increased outreach and advocacy," said Rabun.
+
+Rabun will serve as an employee of the Council on Library Information Resources (CLIR) under terms of an agreement with the IIIF Consortium. She will work at the direction of Michael Keller, chair of the IIIF Consortium Executive Committee, and Tom Cramer, founder and chair of the IIIF Coordinating Committee.
+
+"Sheila's impressive accomplishments in various digital library and scholarly programs coupled with her strong critical thinking, analysis and communication skills will add great capacity for the entire IIIF community," said Keller, who is also the University Librarian at Stanford University. "We look forward to partnering with her to shape this very new, and exciting role in support of improved access to all our digital collections."
+
```

Comment:
Agree that "partnering" seems like a strange word to use, but it's a quote, so...